lib/swagger/responseErrorHandler.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var log = require('../logger');
+
+/**
+ * Swagger response validation middleware
+ *
+ * @param err
+ * @param req
+ * @param res
+ * @param next
+ */
+module.exports = function requestErrorHandler (err, req, res, next) {
+
+    if (err) {
```

Comment:
The indentation in this file is inconsistent with the other files - can we please keep the indentation at 2 spaces?